### PR TITLE
Modify ls to support path arguments

### PR DESCRIPTION
**Pull request by pixelherodev**

### CODE REVIEW

**Comment by pablomayobre** [comment]
Looks like your text editor changed line endings and added a trailing newline at the end of the file. This makes your proposed change hard to parse and is not recommended.

**Comment by pixelherodev** [comment]
Oops. Didn't mean to make that part of this. Reverting...

**Comment by pixelherodev** [comment]
"Looks like your text editor changed line endings and removed the trailing newline at the end of the file. This makes your proposed change hard to parse and is not recommended."
What line endings is it supposed to use?

**Comment by pablomayobre** [comment]
It's using Windows line endings CRLF and no trailing newline

You used LF (unix) line endings and added a trailing newline

**Comment by pixelherodev** [comment]
Is it fixed?

File: OS/DiskOS/Programs/ls.lua [hunk 1]
```diff
@@ -10,4 +18,4 @@ for k, f in ipairs(files) do
   color(fs.isFile(path..f) and 7 or 11)
   print(f.." ",false)
 end
-print("")
+print("")
```
Comment (pablomayobre) [inline OS/DiskOS/Programs/ls.lua]: You still removed the newline at the end of the file, but I would consider this good for merging. At least you fixed most of it (now it's not all red+green hahaha)

**Comment by pablomayobre** [comment]
@RamiLego4Game this PR is good, but I would manually implement the changes and attribute it when committing, since it looks like you have changed line endings on your side too, and there are conflicts.

I don't think that @np511 should bother anymore to keep the PR sane.

The idea is good and simple and it should be considered by the author of the repository

**Comment by Rami-Sabbagh** [comment]
Implemented in fe7ffda830d56e146b6d9314f51dde8a9871d782
